site/src/examples/checkbox/ControlledGroup.tsx

```diff
@@ -0,0 +1,45 @@
+import { ChangeEvent, ReactElement, useState } from "react";
+import { Checkbox, CheckboxGroup } from "@salt-ds/core";
+
+export const ControlledGroup = (): ReactElement => {
+  const checkboxesData = [
+    {
+      label: "Alternatives",
+      value: "alternatives",
+    },
+    {
+      label: "Equities",
+      value: "equities",
+    },
+    {
+      disabled: true,
+      label: "Fixed income",
+      value: "Fixed income",
```

Comment:
value: "fixed-income" maybe?

Reply:
oops this example shouldnt be here - removing all controlled related content